src/main/java/com/mgmtp/radio/service/user/FavoriteSongServiceImpl.java

```diff
@@ -24,6 +25,9 @@ public FavoriteSongServiceImpl(FavoriteSongRepository favoriteSongRepository, Fa
 
 	@Override
 	public Mono<FavoriteSongDTO> create(String userId, FavoriteSongDTO favoriteSongDTO) {
+		if (favoriteSongDTO.getId() == null || favoriteSongDTO.getId().isEmpty()) {
+			favoriteSongDTO.setId(UUID.randomUUID().toString());
+		}
```

Comment:
Why not let mongo do this?

Reply:
how about using "@GeneratedValue" in entity?